sanghyeok/LV_1/가운데-글자-가져오기.js

```diff
@@ -0,0 +1,3 @@
+function solution(s) {
+    return s.length % 2 === 1 ? s[Math.floor(s.length / 2)] : s.substr(s.length / 2 - 1, 2);
```

Comment:
```suggestion
    return s.length % 2 ? s[Math.floor(s.length / 2)] : s.substr(s.length / 2 - 1, 2);
```
코테에서 활용하기 적합한 방법이 아닐 수 있으나, 홀수일 때 나머지가 1(true), 짝수일 때 나머지가 0(false) 인 것을 활용하면 이렇게도 작성하실 수 있습니다. 참고만 하시면 좋을 것 같습니다👍

Reply:
- 오 아주 좋은 방법 입니다 👏
- 하나 더 말씀드리자면 `substr`은 추후 없어질 가능성이 있으니 `substring`을 사용하는것을 추천드립니다!!
- https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/String/substr (MDN 설명)

Reply:
아하 대운님이 말씀하신 활용 방법이 있다는 점을 알게 되었습니다. 참고하겠습니다!!

Reply:
링크 보내 주신 MDN 참고해서 사용하겠습니다 감사합니다!

Reply:
링크 보내 주신 MDN 참고해서 숙지해놓겠습니다! 두분 피드백 감사합니다..!!